NuRadioReco/test/tiny_reconstruction/reference.json

```diff
@@ -1 +1 @@
ull, "cr_azimuth": null, "channels_max_amplitude": 0.2798078137537194, "zenith": 0.7862752532958984, "azimuth": 3.9598489379882813, "zenith_cr_templatefit": null, "zenith_nu_templatefit": null, "cr_xcorrelations": null, "nu_xcorrelations": null, "station_time": null, "cr_energy_em": null, "nu_inttype": null, "chi2_efield_time_direction_fit": null, "ndf_efield_time_direction_fit": null, "cr_xmax": null}, "sim_station_parameters": {"nu_zenith": null, "nu_azimuth": null, "nu_energy": null, "nu_flavor": null, "ccnc": null, "nu_vertex": null, "inelasticity": null, "triggered": null, "cr_energy": 1.58489319246e+18, "cr_zenith": null, "cr_azimuth": null, "channels_max_amplitude": null, "zenith": 0.7853981633974483, "azimuth": 3.957853280977215, "zenith_cr_templatefit": null, "zenith_nu_templatefit": null, "cr_xcorrelations": null, "nu_xcorrelations": null, "station_time": null, "cr_energy_em": 1.3918747974426e+18, "nu_inttype": null, "chi2_efield_time_direction_fit": null, "ndf_efield_time_direction_fit": null, "cr_xmax": 646.2024663}, "channel_parameters": {"zenith": [null, null, null, null], "azimuth": [null, null, null, null], "maximum_amplitude": [0.25234367223241533, 0.27967465467521846, 0.2626114100983252, 0.2798078137537194], "SNR": [{"integrated_power": 0.0, "peak_2_peak_amplitude": 0.004405743307013415, "peak_amplitude": 0.004503107511983823, "Seckel_2_noise": 5}, {"integrated_power": 0.0, "peak_2_peak_amplitude": 0.007432602817755542, "peak_amplitude": 0.007437313104739916, "Seckel_2_noise": 5}, {"integrated_power": 0.0, "peak_2_peak_amplitude": 0.0075361099149315, "peak_amplitude": 0.00777925424682927, "Seckel_2_noise": 5}, {"integrated_power": 0.0, "peak_2_peak_amplitude": 0.004248681946168086, "peak_amplitude": 0.004294274444939337, "Seckel_2_noise": 5}], "maximum_amplitude_envelope": [0.2803517851099761, 0.2802588963924389, 0.26371616474120246, 0.28776646182968296], "P2P_amplitude": [0.5013210481362056, 0.5560835859923348, 0.5205909728639548, 0.551617830513689], "cr_xcorrelations": [null, null, null, null], "nu_xcorrelations": [null, null, null, null], "signal_time": [636.2440214285433, 634.0440214285433, 623.2440214285433, 622.8440214285433]}, "electric_field_parameters": {"ray_path_type": [null, null], "polarization_angle": [1.467057253263737, 1.4890983521441759], "polarization_angle_expectation": [1.324985858091711, -1.816606795498082], "signal_energy_fluence": [[0.0, 3.6371878455679414, 335.5500393638057], [0.0, 2.8768866244517146, 429.10608047739856]], "cr_spectrum_slope": [null, -1.7427483474721464], "zenith": [0.7862752532958984, 0.7862752532958984], "azimuth": [3.9598489379882813, 3.9598489379882813], "signal_time": [574.2189260067184, null], "nu_vertex_distance": [null, null], "nu_viewing_angle": [null, null], "max_amp_antenna": [null, null], "max_amp_antenna_envelope": [null, null], "reflection_coefficient_theta": [null, null], "reflection_coefficient_phi": [null, null], "cr_spectrum_quadratic_term": [null, 0.5690068495080591]}}}
```

Comment:
It is super hard to tell in this json format. @lpyras did you check that the changes were small or explainable by the changes in a consistent way? Please add a general comment so I can get to approving this PR.